.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDateParser.cs

```diff
@@ -470,8 +470,7 @@ private DateTimeResolutionResult ParseDurationWithAgoAndLater(string text, DateO
                 config.DurationParser,
                 config.UnitMap,
                 config.UnitRegex,
-                config.UtilityConfiguration,
-                AgoLaterUtil.AgoLaterMode.Date
```

Comment:
Is removing this correct?

Reply:
yes. Determined the mode in the code before.